src/components/Medicine/MedicineCard.jsx

```diff
@@ -0,0 +1,97 @@
+import React from "react";
```

Comment:
Not really needed here.

Reply:
If I remove this line, it throws the following error:
"MedicineCard.jsx:37 Uncaught ReferenceError: React is not defined at MedicineCard.jsx:37:35" ?

Reply:
Yes, because Line 37 reads:
`export default class Card extends React.Component`
=> React is not defined
=> How about a function component? ;)

```jsx
export default function Card(props) {
    return (
      <div>
        <div>
          <SingleCard id={props.id}>
            <h4>{props.name}</h4>
            <p>{props.einnahmehinweis}</p>
            <CardContent
              einnahmeMorgens={props.einnahmeMorgens}
              einnahmeMittags={props.einnahmeMittags}
              einnahmeAbends={props.einnahmeAbends}
              einnahmeNachts={props.einnahmeNachts}
            />
          </SingleCard>
        </div>
      </div>
    );
}
```

Reply:
This strangely makes sense. :-) Changed it and will focus on re-writing the other functions! Thanks for the help!